src/dropdown.test.forms.ts

```diff
@@ -167,8 +167,9 @@ it('sets the validity message with `setCustomValidity()`', async () => {
   await host.updateComplete;
 
   expect(
-    host.shadowRoot?.querySelector('[data-test="validity-message"]')
-      ?.textContent,
+    host.shadowRoot
+      ?.querySelector('[data-test="validity-message"]')
+      ?.textContent?.trim(),
```

Comment:
Have to trim now because of [this](https://github.com/CrowdStrike/glide-core/pull/1073/files#diff-7431ab16692f7d7a7cfecdb04f55d6c62f87417591065650d88dc407a9b62f3b) formatting change.